### ISSUE TITLE
[Feature Request] : Helper to build tables with separators?

### ISSUE THREAD

**Issue by galenseilis** [spirali/nelsie#50]
As a data scientist, I often need to give tables of values in my presentations.

I looked at some related issues and documentation:
- https://github.com/spirali/nelsie/issues/46
- https://github.com/spirali/nelsie/issues/11
- https://spirali.github.io/elsie/cookbook/grids/

I tried this example (migrated to Nelsie from the Elsie documentation):
```python
@deck.slide()
def another_table(slide):
    row_count = 2
    column_count = 3

    for r in range(row_count):
        row = slide.box(row=True, p_bottom=10)
        for c in range(column_count):
            box = row.box(width=100, height=100, p_right=10)
            box.text(f"({r}, {c})", TextStyle(color="black"))
```

It created a grid of entries, which is a good start!

![image](https://github.com/spirali/nelsie/assets/11385121/4e50d9fe-8e48-4828-855c-2d78129fa9f8)

I would like it to be relatively easy to make something like this (image taken from Google images):
![image](https://github.com/spirali/nelsie/assets/11385121/09d2609d-a327-4a44-815b-8bdd6da4cac2)

Similar to `tabular` in LaTeX, it is possible to declare where line separators exist among the rows and columns.

I'm not sure what the Nelsie-idiomatic way to do this is.

Would it be possible to implement a function that assists in putting together tables with separators?

**Comment by spirali** [comment]
In the current version, there is no better way than you have shown. The problem is that flexbox layout is one-dimensional. I am going to expose the grid layout in the next version of Nelsie, so creating tables should be more easier.

**Comment by spirali** [comment]
Hi,
I have released v0.12.0 that supports grid layout. You can see how to create separators in "bigdemo" example.

**Comment by galenseilis** [comment]
Thank you so much for your efforts!🚀
